### PR TITLE
Fixes #4625 - Adjust input div pos according to position and size changes

### LINKED ISSUE #4625: [Annotations] Long strings disappear if they are pasted into narrow screenshots
[Affected versions]:
- Nightly 63.0a1
- Dev Edition 62.0b7
- Release Unbranded

[Affected Platforms]:
- All Windows
- All Linux
- All Mac

[Prerequisites]:
 - The "xpinstall.signatures.required" boolean pref is set to "false".
 - The "extensions.legacy.enabled" boolean pref is set to "true".
 - The "extensions.install.requireBuiltInCerts " boolean pref is set to "false"
 - Have the Screenshots 33.0.0 dev version installed from [here](https://screenshots.dev.mozaws.net/homepage/install-test-local.html), on a new clean profile.

[Steps to reproduce]:
1. Open the profile from prerequisites and navigate to a website (eg. www.wikipedia.com).
2. Take a narrow screenshot (eg. less than 500p width) and click the "Edit this image" button.
3. Click the "Add text" button.
4. Press the "Ctrl"+"V" key combination and observe the behavior.

[Expected results]:
- The pasted string is in view, contained in the screenshot.

[Actual results]:
- The pasted string disappears from the screen.

[Notes]:
- The textbox starts to move out of sight when pasting strings of 15 characters or longer.
- The textbox disappears completely when pasting strings of 45 characters or longer
- I have attached a screencast of the issue:
ctrl_v_disappear

### PR DESCRIPTION
**Pull request by chenba**

### CODE REVIEW

**Comment by chenba** [comment]
On hold because I need handle the scenario where the user dragged the input off center.

**Comment by chenba** [comment]
@punamdahiya Ready for review.

**Comment by punamdahiya** [comment]
@chenba click on Add Text and pasting a long string using cmd + V has the issue of input string disappearing

File: server/src/pages/shot/text-tool.js [hunk 1]
```diff
@@ -240,15 +238,48 @@ exports.TextTool = class TextTool extends React.Component {
   }
 
   adjustX(e) {
-    if (previousInputText === this.textInput.current.textContent) {
+    // if the text hasn't chagned, do nothing
+    if (this.previousInputText === this.textInput.current.textContent) {
+      console.log("no change gtfo");
```
Comment (punamdahiya) [inline server/src/pages/shot/text-tool.js]: nit: We can remove log before merge

**Comment by punamdahiya** [comment]
@chenba quick question trying PR with handling paste event and fix of #4624 resolve the above issues. Is it supposed to work with #4624 fix?

File: server/src/pages/shot/text-tool.js (hunk 1 above)
Comment (chenba) [reply server/src/pages/shot/text-tool.js]: hah, sorry, that wasn't supposed to be there. :)

**Comment by chenba** [comment]
Simply from looking at the code in the PR for #4624, I don't see how the div's position is being updated any differently than what's in master.  But I'll try it out.

**Comment by chenba** [comment]
#4636 fixed #4625.  Closing this.

**Pull request closed without merging**